tests/unittests/test_request_timeout.py

```diff
@@ -99,7 +99,7 @@ def test_config_provided_request_timeout(self, mock_get, mock_request):
         client = RechargeClient(**config)
         client.request("GET", "dummy_path")
         
-        mock_request.assert_called_with('GET', 'https://api.rechargeapps.com/dummy_path', stream=True, timeout=100.0, headers={'X-Recharge-Access-Token': 'dummy_at', 'Accept': 'application/json', 'User-Agent': 'dummy_ua'})
+        mock_request.assert_called_with('GET', 'https://api.rechargeapps.com/dummy_path', stream=True, timeout=100.0, headers={'X-Recharge-Access-Token': 'dummy_at', 'Accept': 'application/json', 'X-Recharge-Version': '2021-11', 'User-Agent': 'dummy_ua'})
```

Comment:
can we generalise the headers for all the unit tests? it will reduce the line length and improve code readability.

Reply:
@RushiT0122 Updated the test case to code to generalize params.